### PR TITLE
Add brid player video

### PR DESCRIPTION
**Pull request by kul3r4**
#252

### CODE REVIEW

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

File: src/20_Components/amp-brid-player.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  -->
+  <amp-brid-player
+    data-partner="264"
+    data-player="4144"
+    data-video="13663"
+    layout="responsive"
+    width="480" height="270">
+</amp-brid-player>
+  <!--
+    or via the  `data-playlist` attribute. For a complete list of the attributes, refer to the [official documentation](https://github.com/ampproject/amphtml/blob/master/extensions/amp-brid-player/amp-brid-player.md).
```
Comment (sebastianbenz) [inline src/20_Components/amp-brid-player.html]: nit: s/or/... or/

File: src/20_Components/amp-brid-player.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/components/amp-vimeo/">
+  <!-- #### Setup -->
+  <!--
+    Import the `amp-brid-player` component in the header.
+  -->
+  <script async custom-element="amp-vimeo" src="https://cdn.ampproject.org/v0/amp-brid-player-0.1.js"></script>
```
Comment (sebastianbenz) [inline src/20_Components/amp-brid-player.html]: s/amp-vimeo/amp-brid-player/

**Comment by sebastianbenz** [comment]
LGTM few comments

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
LGTM

**Pull request merged**